docs/feature-flags/sdks/node.md

```diff
@@ -46,6 +46,18 @@ await init({
 
 After initialization, the SDK begins polling Eppo’s API at regular intervals to retrieve the most recent experiment configurations such as variation values and traffic allocation. The SDK stores these configurations in memory so that assignments thereafter are effectively instant. If you are using the SDK for experiment assignments, make sure to pass in an assignment logging callback (see [section](#define-an-assignment-logger-experiment-assignment-only) below).
 
+### Initialization options
```

Comment:
Same changes as to `javascript.md` except for no `pollAfterSuccessfulInitialization` option.